File: c/1008-salario.c
#include <stdio.h>

int main(){

  int number, hours;
  double paid, salary;

  scanf("%d", &number);
  scanf("%d", &hours);
  scanf("%lf", &paid);

  salary = hours * paid;

  printf("NUMBER = %d\n", number);
  printf("SALARY = U$ %.2lf\n", salary);

  return 0;
}

File: kotlin/1008-salario.kt
import java.util.*

fun main(args: Array<String>) {
  val NUMBER = readLine()!!.toInt()
  val HOURS = readLine()!!.toInt()
  val PAID = readLine()!!.toDouble()
  val SALARY = HOURS*PAID
  println("NUMBER = $NUMBER")
  println("SALARY = U$ " + "%.2f".format(SALARY))
}
